config/config.js

```diff
@@ -12,7 +12,8 @@ module.exports = {
     operatorsAliases: false
   },
   production: {
-    url: process.env.PRODUCTION_DATABASE_URL,
+    use_env_variable: 'DATABASE_URL',
```

Comment:
Is the `use_env_variable` variable used somewhere?

Reply:
Yes, I saw it as part of settings I need to do to allow heroku run sequelize migrations on heroku postgress.